fiat-github/src/main/java/com/netflix/spinnaker/fiat/roles/github/GitHubProperties.java

```diff
@@ -28,4 +28,6 @@
   @Max(100L)
   @Min(1L)
   Integer paginationValue = 100;
+  @NotNull
+  Integer membershipCacheTTLSeconds = 60 * 1000;
```

Comment:
@ttomsu Thanks for the review! Is this enough to make it configurable? Also should this be a parameter in halyard then?

Reply:
Yeah, we'll need to plumb it through Halyard as well if users want to twiddle the value.